src/main/java/com/backend/dao/Database.java

```diff
@@ -0,0 +1,41 @@
+package com.backend.dao;
```

Comment:
Do you need the `backend` package _and_ the `dao` package?

Reply:
I am using it as a mock database where i am storing the movies list